packages/mobile/src/verify/VerificationEducationScreen.tsx

```diff
@@ -93,6 +119,26 @@ function VerificationEducationScreen({ route, navigation }: Props) {
     setShowLearnMoreDialog(false)
   }
 
+  const showCaptcha = async () => {
+    setIsCaptchaVisible(true)
+    const safetyNetAttestationResponse = await getSafetyNetAttestation()
+    Logger.info('SafetyNet attestation complete:', JSON.stringify(safetyNetAttestationResponse))
+    setSafetyNetAttestation(safetyNetAttestationResponse)
+  }
+  const hideCaptcha = () => setIsCaptchaVisible(false)
+
+  const handleCaptchaResolved = (res: any) => {
+    hideCaptcha()
+    const captchaCode = res?.nativeEvent?.data
+    if (captchaCode !== 'cancel' && captchaCode !== 'error') {
+      Logger.info('Captcha code received: ', captchaCode)
+      // TODO: Add some brief loading indicator and send captcha somewhere before continuing.
```

Comment:
given the feature flag this is okay, but normally i would request some changes 😛

Reply:
Yeah, this PR by itself is not super useful, it's just laying the foundation for the next one when this is integrated with KomenciKit :D